package.json

```diff
@@ -30,7 +30,7 @@
     "audit:public": "npm audit --registry https://registry.npmjs.org/"
   },
   "dependencies": {
-    "@zowe/imperative": "https://github.com:zowe/imperative#json-config-next",
+    "@zowe/imperative": "file:..\\imperative",
```

Comment:
The reason for using Git URLs as the Imperative version numbers was so that our pipelines can find and build a @next version of Imperative.

For example: `git://github.com/zowe/imperative.git#json-config-next`

As documented here: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#git-urls-as-dependencies

Reply:
it makes sense - i couldnt get this to build on my dev machine.  i fixed it in commit.  

im curious how you do this in your development when you need changes to both imperative & zowe cli on your machine.  do you typically do this technique?  wait until imperative is published, then push zowe cli with the latest version of imperative?  something else?

Reply:
Think we discussed this in person 😋
Using NPM v6, I'm able to link Imperative to Zowe CLI on my local machine and build with it.
Perhaps NPM v7 makes the local development process more difficult?

Reply:
yes - thank you for discussing in person.  i think in the end, we'll have to wait until @zowe/imperative@next... is published before we can really get this PR moving forward?